src/MSBuildTaskHost/Concurrent/ConcurrentDictionary.cs

```diff
@@ -0,0 +1,534 @@
+﻿using System;
```

Comment:
@AndyGerlicher Do these classes need a copyright header? They are copied from corefx, so I am not quite sure ...